src/bindings/fluid_cmd.c

```diff
@@ -1969,14 +1969,45 @@ int fluid_handle_basicchannels (void* data, int ac, char** av,
 
 static const char *invalid_arg_msg ="invalid argument\n";
 static const char *too_few_arg_msg = "too few argument, chan mode val [chan mode val]...\n";
+
+/* argument names for channel mode parameter (see resetbasicchannels and 
+  setbasicchannels commands*/
+static const char * name_channel_mode [FLUID_CHANNEL_MODE_LAST]=
+{"poly_omnion","mono_omnion","poly_omnioff","mono_omnioff"};
+
+/*
+  Returns the channel mode num from a name
+  name must be: poly_omnion,  mono_omnion, poly_omnioff, mono_omnioff
+  @name name to search 
+  @ On return: channel mode number (0 to 3) if name is valid, -1 otherwise
+
+*/
+static int get_channel_mode_num(char * name)
+{
+	int i;
+	for (i = 0 ; i <  FLUID_CHANNEL_MODE_LAST;i++)
+	{
+		if ( ! strcmp (name, name_channel_mode[i]))
+		{
+			return i;
+		}
+	}
+	return -1;
+}
+
 /*-----------------------------------------------------------------------------
   resetbasicchannels [chan1 Mode1 nbr1   chan2 Mode2 nbr2 ...]
-  
   Sets the list of MIDI basic channels with mode
+
   This list replaces any previous basic channels list.
 
   With no parameters the function set one channel basic at
   basicchan 0 mode 0 (Omni On Poly) (i.e all the MIDI channel are polyphonic).
+
+  Mode can be: 
+      numeric: 0 to 3 or
+	  name: poly_omnion , mono_omnion, poly_omni_off, mono_omnioff
```

Comment:
Small typo here: poly_omnioff

Reply:
>Small typo here: poly_omnioff.

Thanks for reporting this and probably any others grammar errors.